tests/e2e/test_api.py

```diff
@@ -34,10 +35,14 @@
 from tests.e2e.utils.response_evaluation import ResponseEvaluation
 from tests.scripts.must_gather import must_gather
 
+sys.path.append(Path(__file__).parent.parent.parent.as_posix())
```

Comment:
not clear to me why this is needed?  we are already importing other `test.scripts` content without this

Reply:
it fails in CI If I do not do that for some reason

Reply:
@tisnik maybe you can help understand/explain why this is needed?